internal/podman/client.go

```diff
@@ -24,11 +24,14 @@ import (
 	"strings"
 	"time"
 
+	"github.com/Masterminds/semver/v3"
 	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/log"
 )
 
 var (
-	ErrPodmanNotFound = errors.New("podman not found in your system, check requirements at https://dochub.mongodb.org/core/atlas-cli-deploy-local-reqs")
+	ErrPodmanNotFound           = errors.New("podman not found in your system, check requirements at https://dochub.mongodb.org/core/atlas-cli-deploy-local-reqs")
+	ErrDeterminingPodmanVersion = errors.New("could not determine docker version")
+	minPodmanVersion            = semver.New(5, 2, 0, "", "") //nolint:mnd
```

Comment:
you can check with Ciprian but I think we need to support latest two majors, which would mean v4.9 minimum